client/src/components/Ticketing/checkout/CheckoutPage.tsx

```diff
@@ -58,8 +58,8 @@ export default function CheckoutPage(): ReactElement {
 
   return (
     <div className='bg-zinc-200 flex flex-col md:flex-col sm:flex-col
-         sm:items-center w-full h-full p-20'>
-      <div className='w-full flex flex-row mb-5'>
+         sm:items-center w-full h-full p-2 md:p-20'>
```

Comment:
I think if you change `sm:items-center` to `max-md:items-center` it will fix checkout and the back to events button not being centered below 360px.

Reply:
Display after changing to max-md.  Checkout and back button are left justified now for all screen size.  I think having it left justified looks better also.
![mobile03_after_360px](https://github.com/WonderTix/WonderTix/assets/97830813/3d69a79b-9eb3-4148-9af6-60b8f9e03005)
![mobile06_maxmd](https://github.com/WonderTix/WonderTix/assets/97830813/edfebe08-b782-4050-a3cd-2f0e98b96cd2)